### PR TITLE
minor linting

### PR DESCRIPTION
**Pull request by MartinBernstorff**
Minor type linting. At some point, we want to refactor `evaluate_model`.

### CODE REVIEW

File: src/psycopt2d/utils.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 from collections.abc import MutableMapping
 from pathlib import Path
-from typing import Any, Dict, Iterable, List, Union
+from typing import Any, Iterable, Union, dict, list
 
 import dill as pkl
 import numpy as np
 import pandas as pd
-from omegaconf.dictconfig import DictConfig
+from omegaconf.dictconfig import dictConfig
```
Comment (HLasse) [inline src/psycopt2d/utils.py:9]: does this still work or was it changed by a cmd+f for `Dict` ? :)

File: reports/render_report.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 be in a dir called 'tables' and figures in a dir called 'figs'
 """
 
-from typing import List
+from typing import list
```
Comment (HLasse) [inline reports/render_report.py:9]:
```suggestion
```
can't cmd+f it all ;)

File: src/psycopt2d/evaluate_saved_model_predictions.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 from pathlib import Path
 
 import pandas as pd
-from omegaconf.dictconfig import DictConfig
+from omegaconf.dictconfig import dictConfig
```
Comment (HLasse) [inline src/psycopt2d/evaluate_saved_model_predictions.py:12]: Should probably still be DictConfig

File: src/psycopt2d/feature_transformers.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 """Custom transformers for data preprocessing."""
 from datetime import datetime
-from typing import List, Optional
+from typing import Optional, list
```
Comment (HLasse) [inline src/psycopt2d/feature_transformers.py:3]:
```suggestion
from typing import Optional
```

File: src/psycopt2d/load.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # from datetime import date, datetime, timedelta
 from datetime import date, datetime, timedelta
 from pathlib import Path
-from typing import List, Optional, Union
+from typing import Optional, Union, list
```
Comment (HLasse) [inline src/psycopt2d/load.py:4]:
```suggestion
from typing import Optional, Union
```

File: src/psycopt2d/utils.py [hunk 6]
```diff
@@ -1,12 +1,12 @@
 import time
 from collections.abc import MutableMapping
 from pathlib import Path
-from typing import Any, Dict, Iterable, List, Union
+from typing import Any, Iterable, Union, dict, list
```
Comment (HLasse) [inline src/psycopt2d/utils.py:4]:
```suggestion
from typing import Any, Iterable, Union
```

File: src/psycopt2d/visualization/feature_importance.py [hunk 7]
```diff
@@ -1,4 +1,4 @@
-from typing import Iterable, List, Union
+from typing import Iterable, Union, list
```
Comment (HLasse) [inline src/psycopt2d/visualization/feature_importance.py:1]:
```suggestion
from typing import Iterable, Union
```

File: src/psycopt2d/visualization/performance_over_time.py [hunk 8]
```diff
@@ -3,7 +3,7 @@
 2. AUC by time from first visit
 3. AUC by time until diagnosis
 """
-from typing import Callable, Iterable, List, Optional
+from typing import Callable, Iterable, Optional, list
```
Comment (HLasse) [inline src/psycopt2d/visualization/performance_over_time.py:6]:
```suggestion
from typing import Callable, Iterable, Optional
```

File: src/psycopt2d/visualization/sens_over_time.py [hunk 9]
```diff
@@ -1,5 +1,5 @@
 from functools import partial
-from typing import Iterable, List
+from typing import Iterable, list
```
Comment (HLasse) [inline src/psycopt2d/visualization/sens_over_time.py:2]:
```suggestion
from typing import Iterable
```

**Review by HLasse — COMMENTED** [review]
Approved if my changes fix tests. Remember to run tests before asking for a review ;)

**Comment by MartinBernstorff** [comment]
> Approved if my changes fix tests. Remember to run tests before asking for a review ;) You'll probably also want to change all `dictConfig` back to `DictConfig`

Ah, my apologies! I must've been tired when I requested review here.

File: reports/render_report.py [hunk 10]
```diff
@@ -6,14 +6,13 @@
...
 
-from typing import List
 
 import pandas as pd
 
 from psycopt2d.utils import PROJECT_ROOT
 
 # import pandoc
-# See comment in pyproject.toml on Pandoc, not currently in use. import pandoc
+# See comment in pyproject.toml on Pandoc, not currently in use. Should work now, see: https://github.com/boisgera/pandoc/pull/49#issuecomment-1265983279
```
Comment (HLasse) [inline reports/render_report.py:15]: 🎉

File: reports/render_report.py [hunk 11]
```diff
@@ -34,52 +34,60 @@ def insert_figure(line: str, fig_number: int):
...
-                line = insert_figure(line, fig_number=fig_counter)
-                fig_counter += 1
+                line = insert_figure(line, fig_number=FIG_COUNT)
+                FIG_COUNT += 1
             if "tables" in line:
-                line = insert_table(line, table_number=table_counter)
-                table_counter += 1
-        md.append(line)
+                line = insert_table(line, table_number=TABLE_COUNT)
+                TABLE_COUNT += 1
```
Comment (HLasse) [inline reports/render_report.py:82]: Seems wrong to me to capitalize fig and table counter as they are not constants, but are incremental. Revert?

**Review by HLasse — APPROVED** [review]
Great job powering through this! Minor comment about some constants, otherwise LGTM. Feel free to merge after resolving comments

**Pull request merged**